File: Day 3 : Intro to Conditional Statements/Solution.c
#include <stdio.h>

int main()
{
    int N;
    scanf("%d", &N);
    
    if (N % 2 != 0)
        printf("Weird");
    else
    {
        if (N >= 2 && N <= 5 || N > 20)
            printf("Not Weird");
        else if (N >= 6 && N <= 20)
            printf("Weird");
    }
    return 0;
}

File: Day 3 : Intro to Conditional Statements/Solution.pl
#!/usr/bin/perl

use strict;
use warnings;

my $N = <STDIN>;

if ($N % 2 != 0) {
    print "Weird";
} elsif ($N % 2 == 0 && $N >= 2 && $N <= 5) {
    print "Not Weird";
} elsif ($N % 2 == 0 && $N >= 6 && $N <= 20) {
    print "Weird";
} elsif ($N % 2 == 0 && $N > 20) {
    print "Not Weird";
}